packages/webapp/src/lib/services/RegistryManager.ts

```
@@ -0,0 +1,64 @@
+
+	static setToStorage(value: string): void {
+		try {
+			localStorage.setItem(this.STORAGE_KEY, value);
+		} catch (error) {
+			throw new Error('Failed to save to localStorage: ' + (error instanceof Error ? error.message : String(error)));
+		}
+	}
+
+	static clearFromStorage(): void {
+		try {
+			localStorage.removeItem(this.STORAGE_KEY);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)